src/Service/UserService.php

```diff
@@ -22,6 +22,9 @@ public function __construct(UserRepository $userRepository, UserPasswordEncoderI
     public function newUser(User $user): void
     {
         $user->setPassword($this->encodePassword($user));
+        if (empty($user->getRoles())) {
```

Comment:
this if is not needed anymore? when you create new user you will always add him role